Sources/Services/AWSS3/Sources/AWSS3/Models.swift

```diff
@@ -22522,6 +22523,7 @@ extension GetObjectInput {
                       .withExpiration(value: expiration)
                       .withIdentityResolver(value: config.awsCredentialIdentityResolver, schemeID: "aws.auth#sigv4")
                       .withIdentityResolver(value: config.awsCredentialIdentityResolver, schemeID: "aws.auth#sigv4a")
+                      .withIdentityResolver(value: config.s3ExpressIdentityResolver, schemeID: "aws.auth#sigv4-s3express")
```

Comment:
*[Note: this is generated code]*

For this & several other presigned URLs:
- The client config is added to context so it can be used for the S3 CreateSession operation to get credentials later on.
- The S3 Express identity resolver is added to the operation.